test/controllers/creative_previews_controller_test.rb

```diff
@@ -0,0 +1,8 @@
+require "test_helper"
+
+class CreativePreviewsControllerTest < ActionDispatch::IntegrationTest
+  test "should get show" do
+    get creative_previews_show_url
```

Comment:
These route helpers with `show` `index` etc... always fail. Not sure why the generator is creating them like this? We just need to omit the `show` portion of the method name.